components/dashboard-layout.tsx

```
@@ -0,0 +1,652 @@
+
+        <div className="flex flex-1 flex-col">
+          <header className="sticky top-0 z-10 flex h-16 items-center gap-4 border-b bg-background/80 px-6 backdrop-blur-md">
+            <SidebarTrigger />
+
+            <div className="flex flex-1 items-center gap-4 md:gap-8">
+              <div className="ml-auto flex items-center gap-4 md:gap-6">
+                <ThemeToggle />
+
+                <a
+                  href="https://wa.me/+2348085458632"
+                  target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)